### PR TITLE
 Background mode  and new icons

### PR DESCRIPTION
**Pull request by vesellov**

### CODE REVIEW

**Comment by vesellov** [comment]
only the issue is a conflict in `app/index.js` ... very strange thought...

**Comment by agrishun** [comment]
@vesellov you can merge it

File: app/setup.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        childProcess.stdout.on('error', reject);
+        childProcess.stdout.on('close', resolve);
+    })
+}
+
+const stopBitDust = () => {
+    log.debug('Stopping bitdust');
+	if (process.platform === 'win32') {
+		exec('taskkill  /IM BitDustNode.exe /F /T', options)
```
Comment (vesellov) [inline app/setup.js]: we can also call `BitDustNode.exe stop` here... just need to know absolute path to the file

Comment (vesellov) [reply app/setup.js]: we can do it later... 👍

File: app/setup.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	} else {
+		exec('bitdust stop', options);
```
Comment (vesellov) [inline app/setup.js]: Anton, are you sure `bitdust` alias exists in user's OS ?

**Comment by vesellov** [comment]
getting more and more mature 👍

**Pull request merged**